lib/brain.js

```diff
@@ -3076,10 +3076,11 @@ Brain.prototype._sendCoinsError = function _sendCoinsError (err) {
   const self = this
   if (err.statusCode === INSUFFICIENT_FUNDS_CODE) {
     setTimeout(function () { self._idle() }, self.config.insufficientFundsTimeout)
-    return this._transitionState('insufficientFunds')
+    this._transitionState('insufficientFunds', withdrawFailureRec)
+  } else {
+    this._transitionState('withdrawFailure', withdrawFailureRec)
   }
 
-  this._transitionState('withdrawFailure', withdrawFailureRec)
```

Comment:
Why was this change needed?

Reply:
Probably some leftover from earlier development. Fixed